src/main/java/org/venice/piazza/idam/authz/throttle/JobConsumer.java

```diff
@@ -62,87 +47,32 @@
 	private DatabaseAccessor accessor;
 	@Value("${SPACE}")
 	private String space;
-	@Value("${vcap.services.pz-kafka.credentials.host}")
-	private String kafkaHosts;
-	@Value("#{'${kafka.group}' + '-' + '${SPACE}'}")
-	private String kafkaGroup;
 
-	List<String> topics = new ArrayList<>();
 	private ObjectMapper mapper = new ObjectMapper();
-	private final AtomicBoolean closed = new AtomicBoolean(false);
 	private static final Logger LOGGER = LoggerFactory.getLogger(JobConsumer.class);
 
 	/**
-	 * Initializes consumer listeners for the thread pool workers.
-	 */
-	@PostConstruct
-	public void initialize() {
-		// Get all the Job Type topics that Piazza can fire. This consumer will record each of those actions.
-		List<Class<?>> jobTypes = Arrays.asList(AbortJob.class, IngestJob.class, AccessJob.class, ExecuteServiceJob.class, RepeatJob.class);
-		for (Class<?> jobType : jobTypes) {
-			topics.add(String.format("%s-%s", jobType.getSimpleName(), space));
-		}
-
-		// Start polling for Kafka Jobs on the Group Consumer.
-		// Occurs on a separate Thread to not block Spring.
-		new Thread(() -> pollForJobs()).start();
-	}
-
-	/**
-	 * Opens up a Kafka Consumer to poll for all Jobs that are being processed in the system. This information will be
-	 * recorded in the Throttle table.
+	 * Process a Job Message coming through the Message Bus and determine if it represents a throttable Job
+	 * 
+	 * @param jobTypeString
+	 *            The serialized PiazzaJobType Model
 	 */
-	public void pollForJobs() {
-
-		Consumer<String, String> consumer = null;
-
+	@RabbitListener(queues = { "IngestJob-${SPACE}", "AccessJob-${SPACE}", "ExecuteServiceJob-${SPACE}", "Repeat-Job-${SPACE}" })
```

Comment:
I would pick a naming convention and use it throughout (one has an additional hyphen).